datastore/cloud-client/README.rst.in

```diff
@@ -4,10 +4,16 @@ product:
   name: Google Cloud Datastore
   short_name: Cloud Datastore
   url: https://cloud.google.com/datastore/docs
-  description: > 
+  description: >
     `Google Cloud Datastore`_ is a NoSQL document database built for automatic
     scaling, high performance, and ease of application development.
 
+required_role: Datastore Import Export Admin
+other_required_steps: >
+    Set environment variables:
```

Comment:
Don't you also need to add `Datastore Import Export Admin` and bucket permission to the App Engine default service account?

Reply:
I've added `Datastore Import Export Admin` into the `required_role` section. Pushing tip for `Storage Admin/Owner` roles.

Reply:
There's no mention about App Engine default service account.

@BenWhitehead told me to add the permission to the App Engine default service account.

If it's required, I think we should mention in the README.

Reply:
@IlyaFaer This is the outline of permissions we probably want to callout: https://github.com/GoogleCloudPlatform/python-docs-samples/pull/4121#issuecomment-647767343